vector/v.outlier/v.outlier.html

```diff
@@ -13,12 +13,12 @@ <h2>DESCRIPTION</h2>
 (default), or only positive or only negative outliers. Filtering out
 only positive outliers can be useful to filter out vegetation returns
 (e.g. from forest canopies) from LIDAR point clouds, in order to
-extract Digital Terrain Models. Filtering out only negative outliers
+extract digital terrain models. Filtering out only negative outliers
```

Comment:
If the uppercases were meant to highlight the acronym DTM, why not plug it in for helping searching? Something like 

```suggestion
extract digital terrain models (DTMs). Filtering out only negative outliers
```